### ISSUE TITLE
Onboarding Alejandro Silva

### ISSUE THREAD

**Issue by damianavila** [2i2c-org/team-compass#771]
- Onboarding champion: @damianavila 
- New person's name: Alejandro Silva
- New person's GitHub handle: @aidea775

### Onboarding checklist

The **Onboarding champion** should follow the checklist below to onboard a new team member. Don't hesitate to delegate some actions to others (e.g. if another person needs to give access to an account because you aren't able to).

**First steps**

- [x] Send them a welcome e-mail to introduce yourself and mention that you'll be onboarding them.
- [x] Point them to [our onboarding documentation](https://compass.2i2c.org/en/latest/operations/onboarding.html)

**Accounts**

Create new accounts and share their account name:

- [x] Get member's [GitHub.com account](https://github.com)
- [ ] Get member's [ReadTheDocs.org account](https://readthedocs.org)
- [ ] Get member's [Quay.io account](https://quay.io)
- [ ] Get member's [NameCheap.com account](https://namecheap.com)
- [ ] Get member's [PagerDuty account](https://pagerduty.com)
- [ ] Get member's [bitwarden account](https://bitwarden.com)
- [ ] Get member's [Zoom account](TBD)
- [ ] Get member's [Freshdesk account](TBD)

Create 2i2c accounts and add to team accounts:

- [x] Create a @2i2c.org Google account for them
	- Note that this requires [access to the Admin Panel](https://compass.2i2c.org/administration/google-workspace/#access-and-permissions), a permission held by any 2i2c team lead
- [x] Add them to the proper Google group in our Google Admin space.
- [x] Add to the proper [2i2c GitHub teams](https://github.com/orgs/2i2c-org/teams/)
- [x] Add to the proper [Slack groups](https://2i2c.slack.com/admin/user_groups)
- [ ] Add to the proper [FreshDesk group](https://2i2c.freshdesk.com/a/admin/groups)
- [ ] Add to propoer [PagerDuty Users group](https://2i2c-org.pagerduty.com/users-new)
- [ ] Add to our [bitwarden organization](https://vault.bitwarden.com/#/organizations/11313781-4b83-41a3-9d35-afe200c8e9f1/vault) and add to "Default collection"

**Communication**

- [x] Send them a link to the [2i2c Team Compass](https://compass.2i2c.org/) and its [Team Operations section](https://compass.2i2c.org/operations/)
- [x] Send them a link to the [2i2c calendars and meetings](https://compass.2i2c.org/reference/calendar/)
- [x] Send them a link to the [2i2c Team Google Drive](https://drive.google.com/drive/u/1/folders/0AJcabtB-T0LnUk9PVA)
- [x] Schedule an onboarding meeting for after they've read through the docs

**Documentation**

- [ ] Grant appropriate permissions on [ReadTheDocs projects](https://readthedocs.org/)
  - [ ] https://readthedocs.org/dashboard/2i2c-pilot-hubs/users/
  - [ ] https://readthedocs.org/dashboard/2i2c-team-compass/users/
  - [ ] https://readthedocs.org/dashboard/2i2c-pilot-documentation/users/

**Cloud engineering permissions**

_This is only relevant for others that are joining our Engineering team._

- [x] Add to 2i2c hubs as an admin
- [ ] Add to appropriate GCP organizations and projects
  - [x] Organization `2i2c.org`'s [admin group](https://console.cloud.google.com/iam-admin/groups/03znysh73qbio4n?organizationId=184174754493)
  - [x] Organization `2i2c.org`s [engineering group](https://console.cloud.google.com/iam-admin/groups/01opuj5n2qnifml?organizationId=184174754493)
  - [x] Outside-organization project for [`cloudbank`](https://console.cloud.google.com/iam-admin/iam?project=cb-1003-1696)
  - [x] Outside-organization project for [`meom-ige`](https://console.cloud.google.com/iam-admin/iam?project=meom-ige-cnrs)
  - [x] Outside-organization project for [`callysto`](https://console.cloud.google.com/iam-admin/iam?project=callysto-202316)
  - [ ] (optional) Outside-organization project for [`pangeo-hubs`](https://console.cloud.google.com/iam-admin/iam?project=columbia)
    - An agreement has to be signed first, and they require a copy of our passports, see https://github.com/2i2c-org/meta/issues/247

- [ ] Add to appropriate AWS projects
  - [x] Management account [`2i2c-sandbox`](https://2i2c.awsapps.com/start/#/) (Grants SSO access at https://2i2c.awsapps.com/start#/)
  - [x] Outside-SSO account [`carbonplan`](https://631969445205.signin.aws.amazon.com/console)
  - [x] Outside-SSO account [`openscapes`](https://783616723547.signin.aws.amazon.com/console)
  - [ ] Outside-SSO account [`nasa-veda`](https://smce-veda.signin.aws.amazon.com/console)
    - An agreement has to be signed first, see https://github.com/2i2c-org/leads/issues/104
   
- [ ] Add to appropriate Azure projects
  - [ ] Outside-organization subscription for `utoronto` see https://github.com/2i2c-org/team-compass/issues/386
- [x] Add to [quay.io 2i2c team](https://quay.io/organization/2i2c/teams/owners)
- [ ] NameCheap administration privileges to `2i2c.org` and `2i2c.cloud`

**Roles**

- [ ] Onboard as shadow into the [Support Triager Cycle](https://github.com/2i2c-org/team-compass/issues/new?assignees=&labels=type%3A+onboard&template=new-team-member.md&title=Onboarding+%3Cname%3E)

**External**

- [ ] Add to [2i2c website](https://2i2c.org/organization/)
- [ ] Celebrate their arrival on social media! (Twitter, Mastodon, LinkedIn)

**Wrap up**

- [ ] Have a one-on-one with the new team member to answer any questions they may have
- [ ] Announce in the `#general` Slack channel about our new team member
- [ ] Celebrate, they are now onboarded! 🎉

**Comment by damianavila** [comment]
Created a google account for Alejandro.

**Comment by damianavila** [comment]
Added to the relevant google groups.

**Comment by damianavila** [comment]
Sent invites for the corresponding GH teams.

**Comment by damianavila** [comment]
Sent a link to the 2i2c team drive.

**Comment by jmunroe** [comment]
@damianavila I assume you'll want Alejandro to have access to FreshDesk? If so, I can request another seat licence for him.

**Comment by damianavila** [comment]
> If so, I can request another seat licence for him.

Indeed, @jmunroe, I would appreciate it if you could do it for me 😉, thanks!
I presume buying a seat is a pre-requisite to add it to the group, right?

**Comment by GeorgianaElena** [comment]
Hey @AIDEA775! You should have received a bunch of emails about new accounts on GCP and AWS. You will need to accept the invitations there and change a few passwords after your first login.

Also, can you please create accounts on the platforms below and share the username for each of them once they are ready so I can invite you in the relevant groups? Thanks.

- [ ] Get member's [ReadTheDocs.org account](https://readthedocs.org)
- [ ] Get member's [Quay.io account](https://quay.io)
- [ ] Get member's [NameCheap.com account](https://namecheap.com)
- [ ] Get member's [PagerDuty account](https://pagerduty.com)
- [ ] Get member's [bitwarden account](https://bitwarden.com)
- [ ] Get member's [Zoom account](TBD)
- [ ] Get member's [Freshdesk account](TBD)

**Comment by jmunroe** [comment]
I've created an account on FreshDesk for @AIDEA775 -- he should have an activation email sent to asilva@2i2c.org

**Comment by jmunroe** [comment]
And we have a shared zoom account (see bitwarden for password) across all of 2i2c.

**Comment by AIDEA775** [comment]
I created accounts in the following services with my email at 2i2c:

* RTD (username AIDEA775)
* NameCheap (username alejosilva)
* PagerDuty (email)
* Bitwarden (email)

Thanks you @GeorgianaElena!

**Comment by GeorgianaElena** [comment]
Thank you very much @AIDEA775! You should have received a few more invitations today too.

@damianavila, I believe I covered all of the things that I had privileges to grant access to. Also, I've updated the top comment with the actions that I've took.

**Comment by colliand** [comment]
Welcome @AIDEA775! It's great to have you on the 2i2c team.
